_layouts/default.html

```diff
@@ -175,6 +176,10 @@ <h2>{% if page.breadcrumbs.heading %}{{ page.breadcrumbs.heading }}{% else %}{{
                     <li><a href="https://www.youtube.com/channel/UCpnGCvDSNgxOVrN34eUlS-Q"><i class="fab fa-youtube fa-2x"></i></a><li>
                 </ul>
                 <p class="copyright">Copyright: <span>2020</span> CU Cyber &lt;cyber@clemson.edu&gt;. Design and Developed by <a href="https://themefisher.com">Themefisher</a></p>
+                <br>
+                <ul class="list-inline text-center">
+                    <li><a href="/humans.txt"><img src="/images/humanstxt.gif"></a></li>
+                </ul>
```

Comment:
As I mentioned in my comment, I would favor not adding this or making it a small link in the `p` tag above.

Reply:
rolled back commit 9f83f0a